app/src/main/java/com/isc/hermes/MainActivity.java

```diff
@@ -102,7 +102,7 @@ protected void onCreate(Bundle savedInstanceState) {
         initCurrentLocationController();
         initializeBurgerButtonToolBar();
         initializeFunctionalityOfTheBurgerButton();
-        setTheUserInformationInTheDropMenu();
+        //setTheUserInformationInTheDropMenu();
```

Comment:
Why is this code commented? if some commented code is committed should be have a reason added or just remove commented code